Code/Framework/AzToolsFramework/AzToolsFramework/API/ToolsApplicationAPI.h

```diff
@@ -155,10 +154,19 @@ namespace AzToolsFramework
         virtual void OnEndUndo(const char* /*label*/, bool /*changed*/) {}
 
         /*!
-         * Notify property UI to refresh the property tree.
+         * Notify property UI to refresh the property tree.  Note that this will go out to every
+         * property UI control in every window in the entire application, and using the below function if you can
+         * will yield faster results.
```

Comment:
"the below function" sounds like it's referring to InvalidatePropertyDisplay (the function right below this comment)

```suggestion
         * property UI control in every window in the entire application.
         * Use InvalidatePropertyDisplayForComponent() instead when possible for faster results.
```